spec.md

```diff
@@ -347,6 +353,7 @@ It MUST match the following regular expression:
 ```
 
 The `<length>` is the content-length, in bytes, of the current chunk.
+If the registry provides a `OCI-Chunk-Min-Bytes` header, the size of each chunk, except for the final chunk, SHOULD be greater or equal to that value.
```

Comment:
what happens if the remainder of total/min isn't zero?

Reply:
"except the final chunk" excludes the last chunk. Is there a phrasing change that would help?

Reply:
I added another line to make it clear the last chunk is excluded. Let me know if there's a better phrasing.

Reply:
change looks good to me .. nice and short :-)